src/transports/views.py

```diff
@@ -1,12 +1,34 @@
-from django.shortcuts import render
+import json
 
+from django.shortcuts import render
+from modifications.models import TransportModification
+from transports.models import Transport
 from .forms import TransportForm
 
+from django.views.generic import ListView
+
 # Create your views here.
 def form(request):
-    if request.method == "POST":
-        ...
-        # TODO: vytvorenie/update transportu z formu a trackovanie zmien
+    _form = TransportForm(request.POST or None)
```

Comment:
Toto je fajn poriesene aj pre GET requesty, kazdopadne treba tam pridat instanciu toho objektu, ktoreho id dostaneme v argumente funkcie (id v urlke). Teda ako som spomenul uz v jednom komente keyword argument instance=INSTANCIA_MODELU - https://docs.djangoproject.com/en/3.2/topics/forms/modelforms/ hned navrchu v ukazke kodu posledna ukazka "# Creating a form to change an existing article.".

Je to kvoli tomu, ze ked na forme zavolas .save() tak on vytvori novu instanciu... aj ked upravuje uz nejaku prepravu, on nepozna o aku instanciu sa jedna, teda vytvori novu. My ju v tejto view chceme avsak iba upravit. Cize preto tam davame instance argument. Ak nerozumies, kludne mi pis.